src/modules/logger/messages.h

```diff
@@ -174,31 +216,36 @@ struct ulog_message_logging_tagged_s {
 	char message[128]; //defines the maximum length of a logged message string
 };
 
-struct ulog_message_parameter_header_s {
+/**
+ * @brief Parameter Message
+ *
+ * Includes a parameter value information in the format "<TYPE> <PARAMETER_NAME> <VALUE>"
+ */
+struct ulog_message_parameter_s {
 	uint16_t msg_size;
 	uint8_t msg_type = static_cast<uint8_t>(ULogMessageType::PARAMETER);
 
 	uint8_t key_len;
-	char key[255];
+	char key_value_str[255]; // String with the key and value information
 };
 
 enum class ulog_parameter_default_type_t : uint8_t {
 	system = (1 << 0),
-	current_setup = (1 << 1) //airframe default
+	current_setup = (1 << 1) // Airframe default set by "param set-default <PARAM> <VALUE>" command in the startup script
```

Comment:
This is a bit too specific - it easily gets outdated after the next airframe params refactoring.
```suggestion
	current_setup = (1 << 1) ///< Airframe default
```

Reply:
Thx! I always like providing more context, but it can get outdated 🤔